sdk/webpubsub/azure-messaging-webpubsub/src/main/java/module-info.java

```diff
@@ -1,11 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
 
 module com.azure.messaging.webpubsub {
     requires transitive com.azure.core;
 
-    requires com.nimbusds.jose.jwt;
```

Comment:
@srnagar autorest regeneration caused these packages to be removed. Do you know if this is a bug or if we need to include anything additionally in the generation configurations?

Reply:
The config did not enable `partial-update`.

Though the merge seems still not work after I add it in https://github.com/Azure/azure-sdk-for-java/pull/36805
@haolingdong-msft any idea what happened?

Reply:
@samvaity 

Context for this lib, is that is not a pure DPG lib. It is more a vanilla style, with generated code + handwritten code.

I've also add `disable-client-builder` to config in above PR, to avoid generating the Builder class.

Reply:
https://github.com/Azure/autorest.java/issues/2319#issue-1914650789